libcompat/inet_pton.c

```diff
@@ -83,7 +83,8 @@ inet_pton4(src, dst)
 {
 	static const char digits[] = "0123456789";
 	int saw_digit, octets, ch;
-	unsigned char tmp[NS_INADDRSZ], *tp;
+        unsigned char tmp[NS_INADDRSZ] = 0;
```

Comment:
Array must be initialized with a brace-enclosed initializer, I think
```suggestion
        unsigned char tmp[NS_INADDRSZ] = {0};
```